tools/istio-iptables/main.go

```diff
@@ -62,140 +104,82 @@ func separateV4V6(cidrList string) (NetworkRange, NetworkRange, error) {
 	return ipv4Ranges, ipv6Ranges, nil
 }
 
-func run(args []string, flagSet *flag.FlagSet) {
-
+func constructConfigFromFlags(args []string, flagSet *flag.FlagSet) *Config {
```

Comment:
This signature is misleading. This function is doing 2 things, which should be split into 2 different functions:

- Define flags.
- Build a `Config` from flag values.

Reply:
Yeah. I will remove this method; once I introduce cobra

Reply:
Removed